stencils/pace/stencils/testing/translate.py

```diff
@@ -204,15 +204,11 @@ def slice_output(self, inputs, out_data=None):
                 )
                 for varname, data_element in data_result.items():
                     index = names_4d.index(varname)
-                    if hasattr(data_element, "synchronize"):
-                        data_element.synchronize()
```

Comment:
I believe these were unnecessary, but not sure. There is already a call `utils.device_sync(backend=self.stencil_factory.backend)` above this.

Reply:
Sounds likely.